sdk/core/Azure.Core.Experimental/tests/Variant/StoringArrays.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using NUnit.Framework;
+
+namespace Azure
+{
+    public class StoringArrays
+    {
+        [Test]
+        public void ByteArray()
+        {
+            byte[] b = new byte[10];
+            Value value;
+
+            value = Value.Create(b);
```

Comment:
@KrzysztofCwalina This can have a `MemoryWatch` around it as well. Just didn't do it. You have to add the code path to the `MemoryWatch` static constructor to ensure the JIT is done.